components/browser/storage-sync/src/test/java/mozilla/components/browser/storage/sync/TestUtil.kt

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.storage.sync
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.work.testing.WorkManagerTestInitHelper
+import mozilla.components.service.glean.Glean
+import mozilla.components.service.glean.config.Configuration
+import java.util.Date
+
+/**
+ * Enable testing mode, initialize [WorkManagerTestInitHelper] for testing,
```

Comment:
nit: "This should be called before each test that sends Glean telemetry pings" it's not strictly true: this must be called at least *once* before any test sending or recording glean metrics can work :)